components/invitation/InviteMember.tsx

```diff
@@ -28,9 +28,9 @@ const InviteMember = ({
       role: availableRoles[0].id,
```

Comment:
We have added prettier format to standardise formatting across the source files. Please merge main and then run `npm run format`.